README.md

```diff
@@ -223,7 +223,6 @@ EQ_SERVER_SIDE_STORAGE_DATABASE_URL - url of the database to connect to, e.g. 's
 EQ_SERVER_SIDE_STORAGE_DATABASE_SETUP_RETRY_COUNT - Number of times to retry setting up the database (connection/creation) if it fails
 EQ_SERVER_SIDE_STORAGE_DATABASE_SETUP_RETRY_DELAY_SECONDS - Number of seconds to wait between retry attempts to setup the database
 EQ_GIT_REF - the latest git ref of HEAD on master
-EQ_SR_LOG_GROUP - The name of the log group to create (defaults to `username-local` for local development)
```

Comment:
I already have. Thats what flagged this bit to me 
https://github.com/ONSdigital/eq-terraform/pull/73/files#diff-9c7fec209e9f29aff21eaeecd233420cL206